python/pylibraft/pylibraft/common/mdspan.pxd

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+import pylibraft.common.handle
+
+from cython.operator cimport dereference as deref
+
+
+cdef extern from "raft/thirdparty/mdspan/include/experimental/__p0009_bits/layout_stride.hpp" namespace "std::experimental":  # noqa: E501
```

Comment:
cc @benfred for awareness. This looks very similar to https://github.com/rapidsai/raft/pull/1016 but keeps the `LayoutPolicy`, which will prove to be useful generally.

Reply:
The two `pxd` files are very similar otherwise, so I don't think there's much concern for conflicts across the two.